.devcontainer/.env.sample

```diff
@@ -0,0 +1,17 @@
+###############################################################################
```

Comment:
isn't a file like this already being added in #3556 ? if so you can remove this file for this PR since it doesn't look like it's being used anywhere.